src/cards.ts

```diff
@@ -19,11 +21,33 @@ const sets = CardInfo.reduce(
     {} as Dictionary<Expansion>
 );
 
-const cards: Card[] = CardInfo.map(card => ({
+const validCards = CardInfo.filter(card => (
+    card.set !== "Dark Ages" && 
```

Comment:
extract a constant like `MISSING_SETS`

Reply:
irrelevant, this will change as we would probably need to do a way to select the sets you have